src/Controller/DBHandler.js

```diff
@@ -0,0 +1,450 @@
+import { get, ref, set } from "firebase/database";
+import { database } from "../firebase";
+import axios from "axios";
+
+export default class DBHandler {
```

Comment:
I think the idea behind this DBHandler is nice. Execution is somewhat okay, but I think we could still improve it a bit. The class should be able to handle making DB requests without needing to define the ref in every single function. By right it would be good to use it dynamically, accessing certain data when creating the instance of the class.
`const handler = New DBHandler(ref(db, "string to data here"))` and then you would have a few functions in your handler only:
1. getAll
2. update
3. delete
4. getByID

and maybe others you can think of. However as you might notice, these would be more generic and not so specific to a certain data entity. This would make it easier to use the DB handler and also to extend it.